ecosystem/sep-0010.md

```diff
@@ -250,6 +257,25 @@ A server may choose to issue JWTs for less than all thresholds and based on any
 
 A server that needs to support validating accounts that do not exist can require a signature of the master key of the account address for accounts that do not exist.
 
+### Verifying Client Application Identity
+
+A server may want to attribute each HTTP request to a specific client application. For example, a server and client could make a deal to elimate fees for the users of that client. SEP-10 offers an optional variation to the flow outlined in the [Abstract](#abstract) to enable this use case. 
+
+In order to use this optional feature, the organization that owns the client application must host a [SEP-1 stellar.toml](sep-0001.md) file containing a `SIGNING_KEY` attribute. This allows the SEP-10 server to verify that the challenge sent by a client is signed by the `SIGNING_KEY`, proving that the organization takes ownership of the client application. The SEP-10 server can now attribute requests made with the resulting JWT to the organization that signed the challenge.
+
+1. The client makes the initial `GET <WEB_AUTH_ENDPOINT>` request with an added `?client_domain=example.com` parameter and value
+1. The SEP-10 server generates the challenge transaction with an additional Manage Data operation containing the `<client_domain> attribution>` passed 
```

Comment:
We should add a note here that this behavior continues if the server supports Verifying Client Application Identity. If the server doesn't support it, it should continue with the ordinary flow.

Reply:
I stated this in the parameter definition, but I don't think it needs to be specified in this section, since it's specifically for cases where the parameter is passed.